spec/spec_helper.rb

```diff
@@ -6,10 +6,11 @@
 require 'codeclimate-test-reporter'
 CodeClimate::TestReporter.start
 
+require 'pry'
 require 'rack/test'
 require 'timecop'
-require 'webmock/rspec'
 require 'vcr'
+require 'webmock/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.